syncer/sharding_group.go

```diff
@@ -431,7 +431,10 @@ func (k *ShardingGroupKeeper) AddGroup(targetSchema, targetTable string, sourceI
 	if schemaGroup, ok := k.groups[schemaID]; !ok {
 		k.groups[schemaID] = NewShardingGroup(k.cfg.SourceID, k.shardMetaSchema, k.shardMetaTable, sourceIDs, meta, true)
 	} else {
-		schemaGroup.Merge(sourceIDs)
+		needShardingHandle, synced, remain, err = schemaGroup.Merge(sourceIDs)
```

Comment:
is this a bug

Reply:
Has any side effect with setting `needShardingHandle, synced, remain, err` here?

Reply:
no side effect, but  `_, _, _, err = schemaGroup.Merge(sourceIDs)` seems better?

Reply:
`_, _, _, err = schemaGroup.Merge(sourceIDs)` seems better